### ISSUE TITLE
Vertical farming?

### ISSUE THREAD

**Issue by fabiancook** [nelsonic/learn-gardening#2]
I see a lot of companies using vertical farming to maximise output & quality, is this something your average gardener could do?

**Comment by nelsonic** [comment]
@fabiancook absolutely. Our plan is to grow lots of crops vertically [@home](https://github.com/dwyl/home) and log as much data as possible so we can share it with everyone. 👍
